corehq/apps/locations/resources/v0_1.py

```diff
@@ -63,6 +33,7 @@ class LocationResource(HqBaseResource):
     is_archived = fields.BooleanField(attribute='is_archived', readonly=True)
     can_edit = fields.BooleanField(readonly=True)
     name = fields.CharField(attribute='name', readonly=True, unique=True)
+    have_access_to_parent = fields.BooleanField(readonly=True)
```

Comment:
What is this for?

Reply:
@esoergel To know in the UI that the user is assigned to the ancestor location. For example, a user is assigned to two locations on the same level with the same parent so on this level we don't show 'all' option but if we choose one of these two locations then we show it.